techniques/system/common/1.0/rudder_directives.st

```diff
@@ -0,0 +1,37 @@
+#####################################################################################
+# Copyright 2015 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# This file is the main entry points for the bundle sequence for 
+# Rudder user directives. It is actually a list of method calls, and not
+# a bundle sequence, which allows to add the technique or directives 
+# names as promiser. 
+
+bundle file control {
```

Comment:
you could surround the body file control (this is really a body, not a bundle) by
&if(RUDDER_DIRECTIVES_INPUTS)&